lib/dom/document.js

```diff
@@ -135,6 +135,10 @@
             get: function () { return this._extension; },
             set: function () { throw new Error("Cannot set extension"); }
         },
+        "basename": {
```

Comment:
I'm going to use this convenience getter in Crema for building the document's exported file name.